main.tf

```diff
@@ -113,7 +113,10 @@ resource "aws_iam_policy" "bcgov_perm_boundary" {
         Sid      = "DenyDefaultBudgetAlteration"
       },
       {
-        Action   = "iam:DeleteInstanceProfile"
+        Action = [
+          "iam:DeleteInstanceProfile",
+          "iam:RemoveRoleFromInstanceProfile"
+        ],
```

Comment:
The trailing comma needs to be removed. HCL map items don't require the comma and the JSON encoding ends up incorrect with it.